src/pages/experiments/[experimentId]/data-exploration/index.jsx

```diff
@@ -39,12 +40,22 @@ const ExplorationViewPage = () => {
   const router = useRouter();
   const { experimentId } = router.query;
   const layout = useSelector((state) => state.layout);
+  const embeddingState = useSelector((state) => state.embeddings)
+  const cellSetsState = useSelector((state) => state.cellSets)
```

Comment:
you need to open the ui as a workspace (see my message on slack) so that you get these linting errors auto-corrected

Reply:
Turned out I haven't installed ESLint